src/components/z-popover/index.tsx

```diff
@@ -114,7 +116,7 @@ export class ZPopover {
 
   handleClick(event) {
     this.isVisible ? this.closePopover() : this.openPopover();
-    event.stopPropagation();
+    this.stopEventPropagation && event.stopPropagation();
```

Comment:
io più che aggiungere una prop, avrei lasciato lo stop dell'evento click ma sparato un custom event specifico tipo `triggerClick` con il valore di `this.isVisible` nel detail. mi sembra più pulito e si risparmia una prop